File: api/src/main/java/net/blueshell/api/service/AddressService.java
package net.blueshell.api.service;

import net.blueshell.api.base.BaseModelService;
import net.blueshell.api.model.Address;
import net.blueshell.api.repository.AddressRepository;
import org.springframework.stereotype.Service;

@Service
public class AddressService extends BaseModelService<Address, Long, AddressRepository> {

    public AddressService(AddressRepository repository) {
        super(repository);
    }
}

File: api/src/main/kotlin/net/blueshell/api/service/AddressService.kt
package net.blueshell.api.service

import net.blueshell.api.model.Address
import net.blueshell.api.repository.AddressRepository
import net.blueshell.api.service.base.BaseModelService
import org.springframework.stereotype.Service

@Service
class AddressService(repository: AddressRepository) : BaseModelService<Address, Long, AddressRepository>(repository)